src/LightQuery.Swashbuckle/LightQueryOperationFilter.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Linq;
+using System.Reflection;
+using LightQuery.EntityFrameworkCore;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace LightQuery.Swashbuckle
+{
+    /// <summary>
+    /// Generates operation filter for LightQuery's paramters
+    /// Swashbuckle <see cref="IOperationFilter"/>
+    /// </summary>
+    /// <remarks>OpenAPI Support</remarks>
+    public class LightQueryOperationFilter : IOperationFilter
+    {
+        /*
+         * TODO : how can open api schema be parametric?
```

Comment:
@GeorgDangl I don't know but, People may want to localize or customize **Description** or **Schema Example**

Reply:
I wouldn't wory about localization right now, I think English is fine so far.
If required, developers may hook into the `PostProcess` to alter the schema as they wish.